examples/00-mapdl-examples/bracket_static.py

```diff
@@ -0,0 +1,743 @@
+# value represents pressure into the surface (compressive).
+#
+# To pick the line use the same method used in the previous cell block
+# (:meth:`mapdl.lsel() <ansys.mapdl.core.Mapdl.lsel>`) and then convert the lines
+# to a nodal selection with the :meth:`mapdl.nsel() <ansys.mapdl.core.Mapdl.nsel>`
+# command.
+#
+# Note we have a slightly more complicated picking procedure for the two quarters
+# of the full circle. A method to select the lines would be to select the lower
+# half of the second pinhole circle.
+#
+
+mapdl.lsel("S", "LOC", "Y", pinhole2_Y - pinhole_radius, pinhole2_Y)
+
+###############################################################################
+#
+# And then repick from that selection the lines less than the X center of that pinhole.
+#
+mapdl.lsel("R", "LOC", "X", 0, pinhole2_X)
+
+mapdl.lplot(vtk=True, cpos="xy")
+
+###############################################################################
+#
+# Once we have the correct line we can use the :meth:`mapdl.sf() <ansys.mapdl.core.Mapdl.sf>`
+# command to load the line with a varying surface load.
+#
+
+# Here we load the left side of the lower half of second pin hole
+mapdl.sf("ALL", "PRES", p1, p2)
+mapdl.allsel()
+
+###############################################################################
+#
+# We repeat the procedure for the second pinhole.
+#
+
+mapdl.lsel("S", "LOC", "Y", pinhole2_Y - pinhole_radius, pinhole2_Y)
+mapdl.lsel("R", "LOC", "X", pinhole2_X, pinhole2_X + pinhole_radius)
+
+mapdl.lplot(
+    vtk=True,
+    cpos="xy",
+    show_line_numbering=True,
+)
+
+mapdl.sf("ALL", "PRES", p2, p1)
+mapdl.allsel()
+
+###############################################################################
+# Solution
+# ========
+#
+# To solve an Ansys FE analysis the *Solution* processor needs to be activated,
+# using :class:`mapdl.solution() <ansys.mapdl.core.solution.Solution>`
+# or the APDL :meth:`mapdl.slashsolu() <ansys.mapdl.core.Mapdl.slashsolu>`
+# command. This was done a few steps earlier.
+#
+# The model is ready to be solved using the
+# :meth:`mapdl.solve() <ansys.mapdl.core.Mapdl.solve>` command.
+#
+
+# Solve the model
+output = mapdl.solve()
+print(output)
+
+###############################################################################
+# Mechanical APDL stores the results of this single-load-step problem in the
+# database and in the results file, :file:`Jobname.RST` (or :file:`Jobname.RTH`
+# for thermal, :file:`Jobname.RMG` for magnetic). The database can contain only
+# one set of results at any given time, so in a multiple-load-step or
+# multiple-substep analysis, Mechanical APDL stores only the final solution in
+# the database.
+#
+# Mechanical APDL stores all solutions in the results file.
+#
+# Review the results
+# ==================
+#
+# This step represents the beginning of the postprocessing phase.
+#
+# .. note:: The results you see may vary slightly from what is shown due to variations in the mesh.
```

Comment:
o here you are using the note directive, while earlier you relied on bold formatting for notes. Would you go back and use the note directive for all of them?

Reply:
I might change some to directives, and some leave it as a normal block of text.